src/main/java/seedu/address/model/name/EmployeeName.java

```diff
@@ -10,7 +10,7 @@
 public class EmployeeName {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
```

Comment:
UG states that it can accept alphanumeric characters, as we can't change the regex.

Reply:
noted, will revert